CMakeLists.txt

```diff
@@ -3,64 +3,49 @@ project(UCDD VERSION 0.2.1 LANGUAGES CXX C)
 include(CMakePackageConfigHelpers)
 include(GNUInstallDirs)
 
-option(TESTING OFF)
```

Comment:
Maybe we should change this to `BUILD_TESTING`. It seems like a sort of standard, at least in nixpkgs.

Reply:
Or maybe we should avoid it exactly because it is already used.